content/version/1/4/code-of-conduct.it.md

```diff
@@ -0,0 +1,82 @@
++++
+version = "1.4"
+aliases = ["/version/1/4"]
++++
+
+# Codice di Comportamento del Collaboratore
+
+## Il nostro Impegno
+
+Nell'interesse di promuovere un ambiente aperto e accogliente, noi come
+contributori e manutentori ci impegnamo a partecipare al nostro progetto e
```

Comment:
You missed a bit (“making”) here.
Maybe “ci impegnamo a rendere la partecipazione al nostro progetto e alla nostra ...”